src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -93,6 +93,13 @@ public function getEntitiesByIds($identifier, array $values)
             $values = array_values(array_filter($values, function ($v) {
                 return (string) $v === (string) (int) $v;
             }));
+        } elseif ('guid' === $metadata->getTypeOfField($identifier)) {
+            $parameterType = Connection::PARAM_INT_ARRAY;
```

Comment:
this is wrong. You should not change this to INT_ARRAY. Uuids are not bindable as integers